integration-cli/docker_cli_daemon_test.go

```diff
@@ -1665,6 +1665,8 @@ func (s *DockerDaemonSuite) TestDaemonRestartWithContainerWithRestartPolicyAlway
 }
 
 func (s *DockerDaemonSuite) TestDaemonWideLogConfig(c *check.C) {
+	testRequires(c, x86_64)
```

Comment:
Fixed, the test is passing now, at some point in time this was failing on Power/Z consistently. I agree with you that this test has nothing to do with Architecture.